torch/distributed/_sharding_spec/api.py

```diff
@@ -64,36 +59,26 @@ class ChunkShardingSpec(ShardingSpec):
             The dimension to shard on, could be an integer representing the
             dimension or a string in case of named tensors where dimensions are
             named.
-        placement(List[Device]):
+        placement(List[_RemoteDevice] or List[str]):
             Specifies the placement of each shard of the Tensor. The size of
-            the list represents the number of shards to be created. This
-            parameter can be a list of devices
-            (ex: ["rank:0/cuda:0", "rank:1/cuda:1"]) or a list of custom
-            placement specs.
-
-            The device can be a local device or a remote device specified by one
-            of the following remote formats:
-
-                1. "rank:<rank>/<device>" (ex: "rank:0/cuda:0").
-                2. "<worker_name>/<device>" (ex: "trainer0/cuda:0").
+            the list represents the number of shards to be created. This could
+            be a list of
+            :class:`torch.distributed.remote_device._RemoteDevice`'s. This could
+            also should be a ``List[str]`` where each string represents remote
+            device as accepted by
+            :class:`torch.distributed.remote_device._RemoteDevice`
     """
 
     ShardingDim = Union[int, str]
 
     dim: ShardingDim
-    placements: List[Device]
+    placements: List[_RemoteDevice]
```

Comment:
this seems not matching the doc, it should be `List[Union[_RemoteDevice, str]]`?